### ISSUE TITLE
[Bug]: PLAF203: Notification settings set in PetLibro app reset themselves

### ISSUE THREAD

**Issue by epiphanyplx** [jjjonesjr33/petlibro#42]
### What happened ?

When I try to disable motion of sound notifications in the app it will let me make the change but ignores it. Upon opening again the setting is reset and I continue to get notifications.

I don't see that as an option in the HA integration but was unsure if possibly this integration is unintentionally pushing this setting out/preventing changes to it?
If this isn't something you've seen before I can remove integration and unshare device to see if this resolves, just didn't want to have to readd to HA if there's something that needs to be fixed in the integration.

Thanks for your work on this! Hoping for that camera API!

No relevant log entries when enabling debug logging and attempting change on phone, only see this pertaining to integration:
### Relevant log output

```python

2024-12-10 16:41:45.922 WARNING (ImportExecutor_0) [homeassistant.components.number] custom_components.petlibro.number::PetLibroNumberEntity is overriding deprecated methods on an instance of NumberEntity, this is not valid and will be unsupported from Home Assistant 2022.10. Please report it to the custom integration author
```

**Comment by jjjonesjr33** [comment]
@C4-Dimitri looking at the error that @epiphanyplx is getting do you think this is related to the https://github.com/jjjonesjr33/petlibro/pull/34 pull request?

**Comment by C4-Dimitri** [comment]
Looking at the error message, the error message is related to my pull request, yes.

Looks like i need to swap some naming to the newer formats:
`value` should be `native_value`

Ill open a PR at some point to fix this for all number entities.

I dont think however that the issue they are seeing is related to this error message. I might be wrong.

**Comment by C4-Dimitri** [comment]
@jjjonesjr33, looks like this will be fixed by pr: #43

**Comment by jjjonesjr33** [comment]
Awesome, I'll close out this issue then. 😄
